prow/test/integration/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+package(default_visibility = ["//visibility:public"])
+
+_GO_DEFAULT_TEST = "//prow/test/integration/test:go_default_test"
+
+sh_test(
+    name = "run-test",
```

Comment:
Don't understand the point of this target

Reply:
`--test_output`

Reply:
Sorry I meant dumping logs from the components of prow deployed on kind (crier, sinker, fakeghserver. etc.), would `--test_output` help as well?